Calibration/Hotline/python/hotlineSkims_Output_cff.py

```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
+import copy
+
+from Configuration.EventContent.EventContent_cff import FEVTEventContent
+
+OutALCARECOHotline = cms.PSet(
```

Comment:
... actually ,  #9627 appears in the commit history of this topic branch. Any reason it has to be?